index.html

```diff
@@ -0,0 +1,13 @@
+<html>
```

Comment:
What is the purpose of this file? Seems like it shouldn't be at the root of the project.

Reply:
The index.html file is a temporary work around for for the auth sample app. After logging in we needed someway to redirect back to the sample app's route with the auth token to serve via the webpack-dev-server. The index.html file in the root of the sdk only redirects to back to the app.

Reply:
@adamweeks: the oauth redirect sends us back to localhost, but the app is at `./packages/node_modules/sample-<something>`, so this deals with that without needing to muck with webpack dev server. We'll come up with something better at some point (maybe) - the auth plugin tests do something similar. test-helper-test-server uses the state parameter to figure out which package to redirect back to when the default route gets loaded.

Reply:
@mmaines-cisco `<!DOCTYPE html>`